src/components/skills-quiz/SkillsQuizPage.jsx

```diff
@@ -4,16 +4,18 @@ import { getConfig } from '@edx/frontend-platform/config';
 import { Redirect } from 'react-router-dom';
 
 import SkillsQuiz from './SkillsQuiz';
-import AuthenticatedUserSubsidyPage from '../app/AuthenticatedUserSubsidyPage';
+import AuthenticatedPage from '../app/AuthenticatedPage';
+
+import './styles/SkillsQuizPage.scss';
```

Comment:
I think it's better to import it in [index.scss](https://github.com/edx/frontend-app-learner-portal-enterprise/blob/master/src/index.scss)

Reply:
@saleem-latif My [PR](https://github.com/edx/frontend-app-learner-portal-enterprise/pull/284) is already importing the styles from skill-quiz directory as @zamanafzal suggests. So once the PR is merged, we can discard this statement.